src/views/containers/CreateEvaluation/index.tsx

```diff
@@ -78,6 +78,7 @@ function CreateEvaluation(props: Props) {
       loading={props.loading && loading}
       evaluator={props.evaluator.userName}
       onSubmit={createEvaluation(props.jwt)}
+      reason={this.props.reason}
```

Comment:
Reasons are going to be hard coded. So, create a module (file) name reasons with an object containing reasons for each service and then use it to set the reasons `prop`.
```
// in reasons.tx
basicReasons =  ["inquiry", "claims"];
const reasons = {
 call: basicReasons,
 sms: [...basicReasons, "sms inquiry"]
}
export default reasons;
```

```suggestion
      reason={reasons[props.service]}
```